### PR TITLE
Make sure to reset registry task var when setting up new container

### PR DESCRIPTION
**Pull request by vangheem**

### CODE REVIEW

File: guillotina/contentapi.py [hunk 1]
```diff
@@ -37,6 +37,7 @@ def __init__(self, db, user=RootUser("root")):
         await self.abort()
 
     async def use_container(self, container: IContainer):
+        task_vars.registry.set(None)
```
Comment (lferran) [inline guillotina/contentapi.py]: is the `registry` task var set on `get_registry(container)`?

Comment (lferran) [reply guillotina/contentapi.py]: ah yeah! I see it now :+1:

Comment (vangheem) [reply guillotina/contentapi.py]: Yes, it is! https://github.com/plone/guillotina/blob/master/guillotina/utils/misc.py#L381

File: guillotina/commands/run.py [hunk 2]
```diff
@@ -41,6 +42,10 @@ def get_parser(self):
             return
         sig = inspect.signature(module.run)
         if "container" in sig.parameters:
```
Comment (bloodbare) [inline guillotina/commands/run.py]: I don’t see this related with the issue

**Pull request merged**